packages/main/src/plugin/provider-registry.ts

```
@@ -1632,6 +1632,55 @@ export class ProviderRegistry {
+      this._onDidUpdateVmConnection.fire({
+        providerId,
+        connection: {
+          name: providerConnectionInfo.name,
+          status: (): ProviderConnectionStatus => status,
+          error,
+        },
+        status,
+        error,
+      });
+    }
+  }
```

Biome (2.4.11)

[error] Duplicate class member name "fireConnectionUpdateEvent"

(lint/suspicious/noDuplicateClassMembers)